internal/guestrequest/types.go

```diff
@@ -30,7 +30,12 @@ type LCOWMappedVirtualDisk struct {
 	ReadOnly   bool
 }
 
-// Plan 9.
+type WCOWMappedVirtualDisk struct {
+	ContainerPath string `json:"ContainerPath,omitempty"`
+	Lun           int32  `json:"Lun,omitempty"`
+	AttachOnly    bool   `json:"AttachOnly,omitempty"` //  If `true' then not mapped to the ContainerPath. (Not generated by Swagger)
+}
+
 type LCOWMappedDirectory struct {
```

Comment:
Why not just have a custom file in schema2 called lcow_mapped_virtual_disk.go and have it not be generated but map to LCOW v2? Or put this in LCOW

Reply:
No, I think the best thing (arguably) is to have all the guest request structures in a single package, and keep schema2 to *just* the swagger generated stuff. I don't think it belongs in LCOW as that's more functional than definitions.